buildchain/buildchain/salt_tree.py

```diff
@@ -75,6 +75,7 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
     Path('salt/metalk8s/orchestrate/bootstrap_without_master.sls'),
```

Comment:
Generic comment on the commit messages: please add the issue reference in each commit message